test/browser/renderers.test.tsx

```diff
@@ -116,21 +116,21 @@ function setupContext<R, SR, K extends React.Key>(props: DataGridProps<R, SR, K>
 test('fallback defined using renderers prop with no rows', async () => {
   setup({ columns, rows: noRows, renderers: { noRowsFallback: <NoRowsFallback /> } });
 
-  expect(getRows()).toHaveLength(0);
+  await testLength(getRowsNew(), 1);
```

Comment:
I think using `page.getByRole('row').elements().slice(1)` causes confusion IMO. We can just check the total row count. Also we can await now